ptgnn/neuralmodels/gnn/graphneuralnetwork.py

```diff
@@ -306,16 +394,35 @@ def extend_minibatch_with(
         graph_idx = len(partial_minibatch["num_nodes_per_graph"])
 
         adj_list = partial_minibatch["adjacency_lists"]
+        tensorized_edge_feature_data = partial_minibatch["edge_feature_data"]
         nodes_in_mb_so_far = partial_minibatch["num_nodes_in_mb"]
-        for sample_adj_list_for_edge_type, mb_adj_lists_for_edge_type in zip(
-            tensorized_datapoint.adjacency_lists, adj_list
+
+        all_edge_features = tensorized_datapoint.edge_features
```

Comment:
The naming here is super-confusing: `all_edge_features` is all edge features of the considered datapoint, but `tensorized_edge_feature_data` is all edge features in the minibatch. Maybe rename for clarity?

Reply:
Correct. The next commit will correct the variable name.